backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -94,15 +94,9 @@ public void setDateTestedBackdate(Date date) {
   // - this method is temporary
   // Eventually, this method will be deprecated in favor of getResultSet() and getResultForDisease()
   public TestResult getTestResult() {
-    Hibernate.initialize(this.results);
```

Comment:
we are removing this -- is it because as we were seeing this wasn't doing anything to load the `results` on the `TestOrder`?

Reply:
So after looking into this more this morning, there are two approaches to using `Hibernate.initialize` or Eager/Join Fetch. `Hibernate.initialize` will issue an additional query to fetch the data for results while Eager/Join Fetch will fetch that data in just one query. 

I don't think that result set is a very heavy object so an eager load shouldn't impact performance and we already seem to use result in a lot of places we get TestEvent or TestOrder so we are already fetching that data with a second query.